sdk/rust/oak_config_serializer/src/main.rs

```diff
@@ -78,13 +87,45 @@ impl Default for InitialNodeConfig {
     }
 }
 
+fn get_sha256(data: &[u8]) -> String {
```

Comment:
Add a comment (mostly to clarify that it's already hex encoded)

Reply:
Done